browser-test/browser-test-compose.yml

```diff
@@ -1,67 +1,37 @@
-# Use postgres/example user/passwordcredentials
-version: '3.1'
+# Builds on docker-compose.yml
+version: "3.4"
 
 services:
-
   localstack:
-    image: localstack/localstack
```

Comment:
Where are the images being specified now?  Were these redundant with the "base" yml file?